NURESTConnection.j

```diff
@@ -229,10 +249,21 @@ var NURESTConnectionLastActionTimer,
 
         case NURESTConnectionResponseCodeInternalServerError:
 
-            var errorName        = errorName || @"[CRITICAL] Internal Server Error",
-                errorDescription = errorDescription || @"Please check the log and report this error to the server team";
 
-            [NURESTError postRESTErrorWithName:errorName description:errorDescription connection:aConnection];
+            if ([errorMessages count] > 0)
+            {
+                for (var i = 0; i < [errorMessages count]; i++)
+                    [NURESTError postRESTErrorWithName:errorMessages[i].title
+                                           description:errorMessages[i].description
+                                            connection:aConnection];
+            }
+            else
+            {
+                var errorName        = @"[CRITICAL] Internal Server Error",
+                    errorDescription = @"Please check the log and report this error to the server team";
+
+                [NURESTError postRESTErrorWithName:errorName description:errorDescription connection:aConnection];
+            }
```

Comment:
The original code is:
```
var errorName        = errorName || @"[CRITICAL] Internal Server Error",		
errorDescription = errorDescription || @"Please check the log and report this error to the server team";
```

The if/else replaces the `errorName ||`, since we don't have a single `errorName` anymore